pages/processes-and-artifacts.md

```diff
@@ -207,26 +205,26 @@ change the road you're on:
 
 ## <a name="kickoff"></a>Kick-off meeting
 
-With your [two-pager](#two-pager) in-hand, use it as the official agenda for
-the first kick-off meeting. Ideally you've already had a one-on-one [recruiting
-chat](#recruiting) with most of the people in the room; this ensures that the
-meeting goes quickly, and that other people besides the organizers have a
-chance to speak about some of their own ideas.
+Use your [two-pager](#two-pager) as the official agenda for
+the first kick-off meeting. Ideally, you've already had a one-on-one [recruiting
+chat](#recruiting) with most of the people in the room; this ensures the
+meeting goes quickly and that people other than the organizers have a
+chance to speak about their ideas.
 
 ## <a name="meetings"></a>Weekly meetings
 
 After the warm-fuzzies of the initial [recruiting phase](#recruiting) and the
-[kick-off meeting](#kick-off) have worn off, it's critical to maintain momentum
-and team cohesion by holding brief weekly meetings, ideally only thirty
+[kick-off meeting](#kick-off) have worn off, maintain momentum
+and team cohesion by holding brief weekly meetings, ideally only 30
```

Comment:
So I seem to recall from grade school that numbers below 100 should be spelled out. Did that change in the last 30-35 years?